src/index.html

```diff
@@ -5,10 +5,35 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__container">
+        <a class="header__logo" href="#"> <img src="/src/images/logo.png" alt="logo"></a>
+        <nav class="header__nav nav">
+          <ul class="nav__list">
+            <li class="nav__item"> <a class="nav__link is-active" href="#apple">Apple</a> </li>
+            <li class="nav__item"> <a class="nav__link"  href="#samsung">Samsung</a> </li>
+            <li class="nav__item"> <a class="nav__link" href="#smartphones">Smartphones</a> </li>
+            <li class="nav__item"> <a class="nav__link" data-qa="hover"
+              href="#laptops&computers">Laptops & Computers</a>
+            </li>
+            <li class="nav__item"> <a class="nav__link" href="#gadgets">Gadgets</a> </li>
+            <li class="nav__item"> <a class="nav__link" href="#tablets">Tablets</a> </li>
+            <li class="nav__item"> <a class="nav__link" href="#photo">Photo</a> </li>
+            <li class="nav__item"> <a class="nav__link" href="#video">Video</a> </li>
+          </ul>
+
+        </nav>
+      </div>
+
+    </header>
+
   </body>
```

Comment:
- But don't add them between parent and child elements.
![image](https://github.com/mate-academy/layout_moyo-header/assets/63243136/1620ddf9-ac34-4602-8065-bc38a5c2933b)